PowerEditor/src/ScitillaComponent/AutoCompletion.cpp

```diff
@@ -884,3 +895,15 @@ const TCHAR * AutoCompletion::getApiFileName()
 	return ScintillaEditView::langNames[_curLang].lexerName;
 
 }
+void AutoCompletion::sortKeyWords()
+{
+	if (_ignoreCase) {
+		std::sort(_keyWordArray.begin(), _keyWordArray.end(), [](const auto& lhs, const auto& rhs) {

```

Comment:
FYI:
1. it's possible to create code with VS2015 that also compiles with VS2013.
2. there's a free edition of VS2013.